### PR TITLE
Blog gui challenges switch

### LINKED ISSUE #5821: [CSS] Building a switch component [8/4/2021]
* [Draft](https://docs.google.com/document/d/1PRifS9Qb67Xh_ZX4FA-doP1OFE-KvL-wRS7zh5oOBEo/edit?resourcekey=0-oAnUlmZDv3b6ZheVt6ZWqg#)

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #5821

🔧  todo:
- [x] final reviews
- [x] finalize the date (probably 8/11/21)
- [x] add final youtube embed

### CODE REVIEW

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Some edits, a couple of things to remember for the future (as these were a lot of my changes). 

- Alt text should be a complete sentence. 
- When linking to web.dev, omit the domain.

**Comment by argyleink** [comment]
> Some edits, a couple of things to remember for the future (as these were a lot of my changes).
> 
> * Alt text should be a complete sentence.
> * When linking to web.dev, omit the domain.

thanks Rachel! all the feedback has been integrated and updated ❤️

**Comment by argyleink** [comment]
this post hopes to merge at 9:30am PDT on Wednesday 8/11 to be live and available by 10am. Thanks!

File: src/site/content/en/blog/building-a-switch-component/index.md [hunk 1]
```diff
@@ -0,0 +1,892 @@
...
+Create a demo, [tweet me](https://twitter.com/argyleink) links, and I'll add it
+to the community remixes section below!
+
+### Community remixes
+
+*Nothing to see here yet.*
+
+### Resources
+- `.gui-switch` [source code on
+  Github](https://github.com/argyleink/gui-challenges)
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-a-switch-component/index.md:892]:
```suggestion
### Resources

FInd the `.gui-switch` [source code on
  Github](https://github.com/argyleink/gui-challenges).
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**